### PR TITLE
CIV-5662 : Part admit + already paid- Rejection reason.

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-5662

### Change description ###
This PR has changes to accept the claimant rejection reason.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/services/features/claimantResponse/rejectionReasonService.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import {ClaimantResponse} from '../../../common/models/claimantResponse';
```
Comment (laarmada) [inline src/main/services/features/claimantResponse/rejectionReasonService.ts:1]: Why toy dont use the existing genericService for claimantResponse?

**Review by laarmada — CHANGES_REQUESTED** [review]

File: src/main/services/features/claimantResponse/rejectionReasonService.ts (hunk 1 above)
Comment (laperla11) [reply src/main/services/features/claimantResponse/rejectionReasonService.ts:1]: I agree with Sergio, you can use generic `getClaimantResponse` and `saveClaimantResponse` methods

Comment (neeta-hmcts) [reply src/main/services/features/claimantResponse/rejectionReasonService.ts]: resolved.

File: src/main/common/form/models/claimantResponse/rejectionReason.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {IsNotEmpty} from 'class-validator';
+import {CLAIMANT_REJECTION_RESPONSE_REQUIRED} from '../../../form/validationErrors/errorMessageConstants';
+
+export class RejectionReason {
+  @IsNotEmpty({message: CLAIMANT_REJECTION_RESPONSE_REQUIRED})
```
Comment (commitBlob) [inline src/main/common/form/models/claimantResponse/rejectionReason.ts:5]: This error message will not be translated, please dont use `errorMessageContstants.ts`

**Review by laperla11 — APPROVED** [review]

File: src/main/common/form/models/claimantResponse/rejectionReason.ts (hunk 2 above)
Comment (neeta-hmcts) [reply src/main/common/form/models/claimantResponse/rejectionReason.ts]: resolved

**Review by commitBlob — APPROVED** [review]

**Pull request merged**